doc/plutus-core-spec/untyped-grammar.tex

```diff
@@ -124,3 +128,10 @@ \subsection{Notes}
 the bound variable, so that all $\lambda$-expresssions are of the form
 \texttt{(lam 0 $M$)}; other variables (ie, those not appearing immediately after
 a \texttt{lam} binder) are represented by natural number greater than zero.
+
+\paragraph{Lists in constructor and case terms}
+The grammar defines constructor and case terms to have a variable number of
+subterms written in sequence with no delimiters. This corresponds to the
+concrete syntax, e.g. we write $\constr{0}{t_1\ t_2\ t_3}$. However, in the
+rest of the specification we will abuse notation and treat these terms as
+having \emph{lists} of subterms.
```

Comment:
> it seems that (lam x (case x)) is OK but you can't do anything with it because there are no valid scrutinees.

Yes exactly.

I'll add a high-level paragraph.